Atomic/scan.py

```diff
@@ -91,7 +91,6 @@ def set_scanner():
         if len(self.args.rootfs) == 0:
             scan_list = self._get_scan_list()
             for i in scan_list:
-                i['Id'] = i['Id'].replace(":", "-")
```

Comment:
shouldn't this be left to handle the previous naming?  I think the condition must be changed to replace : only when it is not part of "sha256:"